src/routes/documents/[id]-[slug]/sidebar/Sections.svelte

```diff
@@ -1,28 +1,56 @@
 <script lang="ts">
-  import type { Section } from "$lib/api/types";
+  import type { Document, Note, Section } from "$lib/api/types";
```

Comment:
Both types are unused:

```suggestion
  import type { Document } from "$lib/api/types";
```